src/main/scala/uk/gov/hmrc/selenium/webdriver/DriverFactory.scala

```diff
@@ -182,3 +185,18 @@ class DriverFactory extends LazyLogging {
 }
 
 private case class DriverFactoryException(exception: String) extends RuntimeException(exception)
+
+object DriverFactory {
+  private object BrowserExtensions {
+    lazy val chromiumAccessibilityAssessment: File = {
+      val extractedBrowserExtension = File.createTempFile("chromium-accessibility-assessment", ".crx")
+      extractedBrowserExtension.deleteOnExit()
+      Files.copy(
+        getClass.getResourceAsStream("/browser-extensions/chromium-accessibility-assessment.crx"),
+        extractedBrowserExtension.toPath,
+        StandardCopyOption.REPLACE_EXISTING
+      )
+      extractedBrowserExtension
+    }
+  }
+}
```

Comment:
we have to extract it to an actual location on the filesystem so that chrome can find it - it's not accessible when it's just in the ui-test-runner jar

Reply:
haven't updated the tests yet to account for this change